src/components/cylc/Tree.vue

```diff
@@ -84,6 +82,8 @@ export default {
     },
     onTreeItemExpanded (treeItem) {
       this.expandedCache.add(treeItem)
+      this.expanded = true
```

Comment:
If any item is expanded, we know that the `Tree`'s `expanded` must be set to `true`. This will move the switch back to the off position, letting the user know the "Collapse All" is disabled now.